README.md

```diff
@@ -1,119 +1,9 @@
 # YARC Branding
 
```

Comment:
If you wouldn't mind, a very short description of what this repository is for and when should these branding guidelines be used would be very helpful.

Reply:
Agree on this